tests/cargo-kani/rectangle-example/Cargo.toml

```diff
@@ -8,5 +8,8 @@ edition = "2018"
 
 [dependencies]
 
-[dev-dependencies]
-proptest = "1.0.0"
+# Turned off while proptest integration is merging. The first PR will
+# not support any integers, which crashes this test case. See
+# https://github.com/model-checking/kani/issues/1473 for details.
+# [dev-dependencies]
+# proptest = "1.0.0"
```

Comment:
I prefer if you add a runtime option for users to enable proptest analysis.

Reply:
I'll resolve this along with the comment with `IS_KANI_PROPTEST`